docs/application-connector/docs/08-04-register-secured-api.md

```diff
@@ -51,3 +53,46 @@ This is an example of the `api` section of the request body for an API secured w
             },
         }  
 ```
+
+## Register an API secured with generated client certificates
```

Comment:
The API is secured with a client certificate. We are just generating example one.
You can describe it as the additional feature but leave main message as `Register an API secured with a client certificate`